templates/solaris-0.1.2/algorithmia.conf

```diff
@@ -0,0 +1,5 @@
+{
+    "username": "__USER__",
+    "algoname": "__ALGO__",
+    "language": "anaconda3x"
```

Comment:
should we change the language here? the package requires python 36, is `anaconda3x-python36` reasonable?

Reply:
This should match a real language so what you have no seems good to me